openslides/users/apps.py

```diff
@@ -29,5 +29,5 @@ def ready(self):
         register_slide_model(User, 'participant/user_slide.html')
 
         # Register viewsets.
-        router.register('users/user', UserViewSet)
-        router.register('users/group', GroupViewSet)
+        router.register('user/user', UserViewSet)
```

Comment:
Are you sure? I can't remember. In this case we would name the app "agenda" to "agendas" and what would be the name for "core"?